flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamNode.java

```diff
@@ -336,12 +336,11 @@ public void setUserHash(String userHash) {
 		this.userHash = userHash;
 	}
 
-	@VisibleForTesting
 	public void setSortedInputs(boolean sortedInputs) {
 		this.sortedInputs = sortedInputs;
 	}
 
-	boolean getSortedInputs() {
+	public boolean getSortedInputs() {
```

Comment:
I think this change is not needed anymore.

Reply:
I made this change because I found it strange that `setSortedInputs` (which is a more concerning method) is `public` and the corresponding getter is in the default scope. 

The setter must be `public` because it is used in `BatchExecutionUtils` in `org.apache.flink.streaming.runtime.translators` package.